task/buildah/0.2/buildah.yaml

```diff
@@ -360,6 +385,7 @@ spec:
       unshare -Uf $UNSHARE_ARGS --keep-caps -r --map-users 1,1,65536 --map-groups 1,1,65536 -w ${SOURCE_CODE_DIR}/$CONTEXT -- buildah build \
         $VOLUME_MOUNTS \
         "${BUILDAH_ARGS[@]}" \
+        "${DEFAULT_LABELS[@]}" \
```

Comment:
Should this be the `LABELS` array? (And we should prepend `DEFAULT_LABELS` to it)

Reply:
Or just pass both `"${DEFAULT_LABELS[@]}" "${LABELS[@]}" \` in this order

Reply:
Revised in b978eda5, for clarity.